gef.py

```diff
@@ -4834,6 +4834,59 @@ def import_structures(self, structs):
         return
 
 
+@register_command
+class ScanSectionCommand(GenericCommand):
+    """Search for addresses belonging to one mapping that are located in another."""
+
+    _cmdline_ = "scan"
+    _syntax_  = "{:s} BELONG LOCATION".format(_cmdline_)
```

Comment:
I think BELONG and LOCATION are weird names for the args.

Maybe something like section and target? Can you think of something else?

Reply:
Naming things is hard. I wasn’t too happy with them but the others sounded ambiguous. I’ll have another go